test/conformance/gateway_conformance_test.go

```diff
@@ -39,7 +42,7 @@ var skippedTestsForExpressionRoutes = []string{
 	tests.HTTPRouteRedirectPortAndScheme.ShortName,
 	// https://github.com/Kong/kubernetes-ingress-controller/issues/3680
 	tests.GatewayClassObservedGenerationBump.ShortName,
-	// https://github.com/Kong/kubernetes-ingress-controller/issues/3678
+	// https://github.com/Kong/kubernetes-ingress-controller/issues/4312
```

Comment:
What's the remaining reason for omitting this test until we have expression route support? Can we add a clarifying comment as to why?

I thought that was only necessary when we needed to manage rule or cross-Route priority order, but https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/tests/tlsroute-simple-same-namespace.yaml only has the one single-rule TLSRoute.

Reply:
The remaining reason for this test is that expression routes do not support L4 routes yet. #4312 is about it.

Reply:
:facepalm: I only just now noticed we had a separate `skippedTestsForExpressionRoutes` array and thought there was only the one.